portfolio/src/main/webapp/electronics_info.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The info for each clickable item in the picture
+const info = {
+  tackleBox: 'A cheap fishing tackle-box I bought from a hardware store.\
+      They are the ideal way to organize your small components.',
+  ti81: 'A Ti-81 calculator I found at a thrift store. The LCD screen\
```

Comment:
Nit: I believe this is TI-81 (capital I)

Reply:
You're absolutely correct! Fix pushed.